build.sbt

```diff
@@ -2,85 +2,96 @@ import Dependencies._
 import xerial.sbt.Sonatype._
 
 lazy val qbeastCore = (project in file("core"))
-  .settings(name := "qbeast-core", libraryDependencies ++= Seq(apacheCommons % Test))
+  .settings(
+    name := "qbeast-core",
+    version := "0.1.0",
+    commonSettings,
+    releaseSettings,
+    libraryDependencies ++= Seq(apacheCommons % Test))
+
+lazy val qbeastSparkVersion = "0.2.0"
 
 // Projects
 lazy val qbeastSpark = (project in file("."))
   .enablePlugins(ScalaUnidocPlugin)
   .dependsOn(qbeastCore)
   .settings(
     name := "qbeast-spark",
+    version := qbeastSparkVersion,
+    commonSettings,
     libraryDependencies ++= Seq(
+      qbeastCoreDep,
+      deltaCore % Provided,
       sparkCore % Provided,
       sparkSql % Provided,
       hadoopClient % Provided,
-      deltaCore % Provided,
+      sparkFastTests % Test,
       amazonAws % Test,
       hadoopCommons % Test,
       hadoopAws % Test),
+    noWarningInConsole,
+    Compile / doc / scalacOptions ++= Seq(
+      "-doc-title",
+      "qbeast-spark",
+      "-doc-version",
+      qbeastSparkVersion,
+      "-doc-footer",
+      "Copyright 2022 Qbeast - Docs for version " + qbeastSparkVersion + " of qbeast-spark"),
     Test / parallelExecution := false,
     assembly / test := {},
     assembly / assemblyOption := (assembly / assemblyOption).value.copy(includeScala = false),
     publish / skip := true)
-  .settings(noWarningInConsole)
-
-qbeastSpark / Compile / doc / scalacOptions ++= Seq(
-  "-doc-title",
-  "qbeast-spark",
-  "-doc-version",
-  qbeast_spark_version,
-  "-doc-footer",
-  "Copyright 2022 Qbeast - Docs for version " + qbeast_spark_version + " of qbeast-spark")
 
 lazy val qbeastSparkNodep = (project in file("nodep"))
-  .settings(name := "qbeast-spark-nodep", Compile / packageBin := (qbeastSpark / assembly).value)
+  .settings(
+    name := "qbeast-spark-nodep",
+    version := qbeastSparkVersion,
+    commonSettings,
+    publishGithubSettings,
+    Compile / packageBin := (qbeastSpark / assembly).value)
 
-// As root project has publish / skip := true, we need to create a wrapper project to publish on
-// sonatype and Maven Central, which has "qbeast-spark" as name.
 lazy val qbeastSparkMaven = (project in file("maven"))
-  .settings(name := "qbeast-spark", Compile / packageBin := (qbeastSpark / assembly).value)
-
-// Common metadata
-val qbeast_spark_version = "0.2.0"
-ThisBuild / version := qbeast_spark_version
-ThisBuild / organization := "io.qbeast"
-ThisBuild / organizationName := "Qbeast Analytics, S.L."
-ThisBuild / organizationHomepage := Some(url("https://qbeast.io/"))
-ThisBuild / startYear := Some(2021)
-ThisBuild / libraryDependencies ++= Seq(
-  fasterxml % Provided,
-  sparkFastTests % Test,
-  scalaTest % Test,
-  mockito % Test)
-
-Test / javaOptions += "-Xmx2G"
-Test / fork := true
-
+  .settings(
```

Comment:
we can delete this qbeastSparkMaven task now, we can publish directly from the tasks.